newsroom/navigations/utils.py

```diff
@@ -17,13 +17,15 @@ async def get_navigations(user: UserData | None, company: Company | None, produc
     """
     Returns list of navigations for given user and company
     """
+    from newsroom.products import get_products_by_company
```

Comment:
@eos87 Is this required here inside the function? As we're already importing from `newsroom.products` at the top of this function, that tells me there shouldn't be a cyclic import issue

Reply:
Actually, there is a cyclic import. If you see it again, the top import is `newsroom.products.products` and the one that has the cyclic import is `newsroom.products` (meaning `__init__.py`). Maybe I can double check what is producing the issue but I didn't want to waste too much time on that at the given time as there were some trickier things to deal with.

Reply:
Yeah fair enough, all good then